cirq-core/requirements.txt

```diff
@@ -2,6 +2,7 @@
 
 duet>=0.2.8
 matplotlib~=3.0
+mpmath~=1.3.0
```

Comment:
Thank you for tracking this down!

I think we can use a less restrictive mpmath spec to be in sync with sympy requirement for [mpmath>=0.19](https://github.com/sympy/sympy/blob/24792d5f4fbf8fe4365101e53b735c6c1f140e0f/setup.py#L364).

```suggestion
# Temporary fix for https://github.com/sympy/sympy/issues/26273
# TODO: remove once `pip install --pre sympy` works in a fresh environment
mpmath<=1.3.*
```

Reply:
@pavoljuhas 
Thank you for comment!
I have reflected the comment and confirmed that it passes the test.